third_party/silabs/efr32_sdk.gni

```diff
@@ -130,6 +130,7 @@ template("efr32_sdk") {
     # Treat these includes as system includes, so warnings in them are not fatal.
     _include_dirs = [
       "${efr32_sdk_root}",
+      "${examples_plat_dir}",
```

Comment:
This is strange and is not part of the efr32_sdk. what are we including this here?

Reply:
we are trying to include spi_multiplex.h from that location

Reply:
Maybe spi_multiplex.h and related code should not be in `examples_plat_dir` if it is needed by SDK files.

the dependency should be the other way around (examples_plat_dir sources requiring the SDK. not SDK requiring examples_plat_dir)

Please add a TODO comment explain why it is included three and that it need to be fixed/cleaned

Reply:
Added TODO